ts/src/hitbtc.ts

```diff
@@ -2369,6 +2369,82 @@ export default class hitbtc extends Exchange {
         }, market);
     }
 
+    async fetchMarginMode (symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name hitbtc#fetchMarginMode
+         * @description fetches margin mode of the user
+         * @param {string} symbol unified symbol of the market the order was made in
+         * @param {object} [params] extra parameters specific to the hitbtc api endpoint
+         * @returns {string} 'cross' or 'isolated'
```

Comment:
instead of returning a string, I think we could return the object with symbol and `marginMode`

Reply:
@carlosmiei I added MarginMode type in ts, are ts types also transpiled?

Reply:
@sc0Vu more or less, but I will check out